cue/load/tags.go

```diff
@@ -88,6 +88,11 @@ func DefaultTagVars() map[string]TagVar {
 				return ast.NewLit(token.INT, string(hx[:])), nil
 			},
 		},
+		"arch": {
+			Func: func() (ast.Expr, error) {
+				return ast.NewString(runtime.GOARCH), nil
+			},
+		},
```

Comment:
nit: place it next to `os`, just like in the docs?

Reply:
@mvdan I deliberately played it safe, just in *case* someone out there is relying on the "ordering" of the struct that `DefaultTagVars()` returns. I'm aware that this is an utter corner case, probably meaningless given the unordered nature of the return type, and no-one **should** be doing that - but that was the motivation behind not placing it next to `os`.

Thoughts?

Reply:
Maps in Go are entirely unordered, so it is literally the same no matter what order you put the keys in :)

Reply:
Fixed in the most recent force-push.